spaceship.zsh

```diff
@@ -407,16 +407,16 @@ spaceship_docker_version() {
 
   command -v docker > /dev/null 2>&1 || return
 
-  if [[ -z $DOCKER_MACHINE_NAME ]]; then
-    DOCKER_MACHINE_NAME="localhost"
-  fi
-  
   local docker_version=$(docker version -f "{{.Server.Version}}")
 
   [[ ${SPACESHIP_PREFIX_SHOW} == true ]] && echo -n "%B${SPACESHIP_PREFIX_DOCKER}%b" || echo -n ' '
 
   echo -n "%{$fg_bold[cyan]%}"
-  echo -n "${SPACESHIP_DOCKER_SYMBOL}  v${docker_version} via〔$DOCKER_MACHINE_NAME〕"
+  if [[ -z $DOCKER_MACHINE_NAME ]]; then
+    echo -n "${SPACESHIP_DOCKER_SYMBOL}  v${docker_version}"
+  else
+    echo -n "${SPACESHIP_DOCKER_SYMBOL}  v${docker_version} via〔$DOCKER_MACHINE_NAME〕"
```

Comment:
There's not, actually. It's just a weird bracket character which leaves a lot of padding.